cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -146,29 +146,29 @@ Hadoop/HBase Environment
 
 For a distributed enterprise, you must install these Hadoop components:
 
-+---------------+-------------------+--------------------------------------------------+
-| Component     | Source            | Required Version                                 |
-+===============+===================+==================================================+
-| **HDFS**      | Apache Hadoop     | 2.0.2-alpha through 2.5.0                        |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
-| **YARN**      | Apache Hadoop     | 2.0.2-alpha through 2.5.0                        |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
-| **HBase**     | Apache            | 0.94.2+, 0.96.x, and 0.98.x                      |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
-| **Zookeeper** | Apache            | Version 3.4.3 through 3.4.5                      |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
-| **Hive**      | Apache            | Version 0.12.0 through 0.13.1                    |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.3.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
++---------------+-------------------+----------------------------------------------------+
+| Component     | Source            | Required Version                                   |
++===============+===================+====================================================+
+| **HDFS**      | Apache Hadoop     | 2.0.2-alpha through 2.5.0                          |
++               +-------------------+----------------------------------------------------+
+|               | CDH or HDP        | (CDH) 5.0.0 through 5.4.4 or (HDP) 2.0, 2.1 or 2.2 |
```

Comment:
(HDP) 2.0, 2.1, or 2.2 
[I hate to ask, as I know adding this comma is a pain to change.]